articles/client-platforms/cyclejs/01-login.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Login
+description: This tutorial will show you how to use the Auth0 Cycle.js driver to add authentication and authorization to your web app.
+---
+
+## Cycle.js Tutorial
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* NodeJS 6.3
+* npm 3.10.3
+* @cycle/xstream-run 3.0.3
+* cyclejs-auth0 1.1.0
+:::
+
+${include('../\_callback')}
+
+### 1. Install the cyclejs-auth0 package
+
+To install both `cyclejs-auth0` and the `cyclic-router` (needed to parse the token sent by Auth0), from the command line, run:
+
+${snippet(meta.snippets.dependencies)}
+
+### 2. Instantiate the driver and configure the Auth0Lock
+
+In your main application file, you can now setup the `auth0Driver` and feed it you `client-ID` and `domain`:
+
+${snippet(meta.snippets.setup)}
+
+### 3. Implement the login
+
+Now that everything is setted, you can activate authentication on some of your components. Activating authentication on a component is a simple as calling the `protect` function on that component.  
```

Comment:
`...everything is setted...` => `...everything is set...`
`...is a simple...` => `...is as simple...`